src/target/linux.rs

```diff
@@ -218,6 +277,103 @@ impl LinuxTarget {
 
         Ok(tasks)
     }
+
+    #[cfg(target_arch = "x86_64")]
```

Comment:
Please make this function available on all platforms and instead return an error on unsupported platforms.